tests/upgrades/test_virtwho.py

```diff
@@ -78,6 +78,9 @@ def test_pre_create_virt_who_configuration(
             3. Report is sent to satellite.
             4. Virtual sku can be generated and attached.
         """
+        org = target_sat.api.Organization().search(query={'search': f'name={ORG_DATA["name"]}'})
+        if org:
+            target_sat.api.Organization(id=org[0].id).delete()
```

Comment:
Another approach would be passing the dynamic org as part of pre_upgrade and adding that part of save_data. Non-blocking though

Reply:
> Another approach would be passing the dynamic org as part of pre_upgrade and adding that part of save_data. Non-blocking though

@omkarkhatavkar when use dynamic name 
```
ORG_DATA = {'name':gen_string('alpha')}
```
the post-upgrade will also use another dynamic name, this is not the same as the pre-upgrade case.
What is your suggestion?

Reply:
you can pass the dynamic org name in save_data function, fetch that same name in post upgrade

Reply:
@omkarkhatavkar Have already use the dynamic name and save_test_data to realize this function, thanks for your help.